rust/repositories.bzl

```diff
@@ -34,6 +34,7 @@ def rust_repositories(
         edition = None,
         dev_components = False,
         sha256s = None,
+        experimental_rust_analyzer_support = False,
```

Comment:
Do we consider `rust_analyzer` experimental? But also I feel like this could be `include_rustc_srcs` and we comment that it's useful for when you're using rust analyzer.